src/adapters/lexicons/adapter.js

```diff
@@ -7,23 +7,123 @@ import DefaultConfig from '@/adapters/lexicons/config.json'
 let cachedDefinitions = new Map()
 
 class AlpheiosLexiconsAdapter extends BaseAdapter {
+  /**
+  * Lexicons adapter uploads config data, defines default options and inits data
+  * @param {Object} config - properties with higher priority
+  */
   constructor (config = {}) {
     super()
     this.config = this.uploadConfig(config, DefaultConfig)
     this.options = { timeout: this.config.timeout ? this.config.timeout : 0 }
-    this.data = null
   }
 
+  /**
+  * This method retrieves short definitions for given homonym
+  * @param {Homonym} homonym - homonym for retrieving definitions
+  * @param {Object} options - options
+  * @return {Boolean} - result of fetching
+  */
   async fetchShortDefs (homonym, options = {}) {
-    let res = await this.fetchDefinitions(homonym, options, 'short')
-    return res
+    await this.fetchDefinitions(homonym, options, 'short')
   }
 
+  /**
+  * This method retrieves full definitions for given homonym
+  * @param {Homonym} homonym - homonym for retrieving definitions
+  * @param {Object} options - options
+  * @return {Boolean} - result of fetching
```

Comment:
Same here (return type)

Reply:
Fixed (missed to fix comments after last updates of methods - thanks for notice!)